src/manager/node.c

```diff
@@ -1321,6 +1325,54 @@ static int node_method_set_unit_properties(sd_bus_message *m, void *userdata, UN
         return 1;
 }
 
+static int passthrough_node_method_callback(AgentRequest *req, sd_bus_message *m, UNUSED sd_bus_error *ret_error) {
+        sd_bus_message *request_message = req->userdata;
+
+        if (sd_bus_message_is_method_error(m, NULL)) {
+                /* Forward error */
```

Comment:
Nit-picky: I think this comment could be removed.

Reply:
Done